src-plugins/manualRegistration/manualRegistration.h

```diff
@@ -19,6 +19,11 @@
 
 class manualRegistrationPrivate;
 
+namespace TransformName
+{
+    enum Value {RIGID,AFFINE};
+}
```

Comment:
(and instead of the namespace you can place the enum in a class to use its namespace.